lightning/src/events/mod.rs

```diff
@@ -1102,11 +1102,21 @@ pub enum Event {
 		///
 		/// May contain a closed channel if the HTLC sent along the path was fulfilled on chain.
 		path: Path,
-		/// The hold times as reported by each hop. The unit in which the hold times are expressed are 100's of
-		/// milliseconds. So a hop reporting 2 is a hold time that corresponds to roughly 200 milliseconds. As earlier
-		/// hops hold on to an HTLC for longer, the hold times in the list are expected to decrease. When our peer
-		/// didn't provide attribution data, the list is empty. The same applies to HTLCs that were resolved onchain.
-		/// Because of unavailability of hold times, the list may be shorter than the number of hops in the path.
+		/// The time that each hop indicated it held the HTLC.
+		///
+		/// We expect that at each hop the hold time will be strictly greater than the hold time of
```

Comment:
The unrounded hold times should be strictly greater, but clearly after rounding that may not be the case anymore. Not sure if that's fully clear now.